backends/ebpf/runtime/ebpf_kernel.h

```diff
@@ -39,6 +39,9 @@ limitations under the License.
 
 #define load_byte(data, b) (*(((u8*)(data)) + (b)))
 #define load_half(data, b) bpf_ntohs(*(u16 *)((u8*)(data) + (b)))
+#define load_half_ne(data, b) (*(u16 *)((u8*)(data) + (b)))
+#define load_word_ne(data, b) (*(u32 *)((u8*)(data) + (b)))
+#define load_dword_ne(data, b) (*(u64 *)((u8*)(data) + (b)))
```

Comment:
could move this set to below with a comment (e.g load without endian conversion)

Reply:
okay